libsolutil/Algorithms.h

```diff
@@ -79,9 +79,6 @@ class CycleDetector
 /**
  * Generic breadth first search.
  *
- * Note that V needs to be a comparable value type. If it is not, use a pointer type,
- * but note that this might lead to non-deterministic traversal.
```

Comment:
The first sentence is still true - you'll still need a comparison operator in order to be able to put stuff in ``visited``...

Reply:
true